README.rst

```diff
@@ -101,7 +79,7 @@ Check the `example`_ folder for some sample usage!
     environment variables.
 
     Variables are shallowly resolved once, then anything in
-    {`...`} is eval'd in Python.
+    {%...%} is eval'd in Python.
```

Comment:
Not quite - only supported [control structures](http://jinja.pocoo.org/docs/dev/templates/#list-of-control-structures) go in `{% ... %}`. Shallow evaluation is going away because Jinja2 allows [basic expressions](http://jinja.pocoo.org/docs/dev/templates/#expressions) in `{{ ... }}`, eliminating the necessity of stuff like dict accessor variables (`irssi_secondary_n` and the like).